dll/win32/shell32/utils.cpp

```diff
@@ -483,6 +483,200 @@ Int64GetNumFormat(
     }
 }
 
+/*************************************************************************
+ *                RealShellExecuteExA (SHELL32.266)
+ */
+EXTERN_C
+HINSTANCE WINAPI
+RealShellExecuteExA(
```

Comment:
I would strongly suggest you move these functions to where the other ShellExecute(Ex) functions are implemented, that is, within shlexec.cpp instead.

Reply:
Moved.